bin/straxer

```diff
@@ -112,13 +122,20 @@ def main(args):
     print(f"\tstrax {strax.__version__} at {osp.dirname(strax.__file__)}")
     import straxen
     print(f"\tstraxen {straxen.__version__} at {osp.dirname(straxen.__file__)}")
-
-    st = getattr(straxen.contexts, args.context)()
+    if not args.package == 'straxen':
+        # pylint: disable=exec-used
+        exec(f'import {args.package}')
+    module = getattr(locals()[args.package], 'contexts')
+    st = getattr(module, args.context)()
     if args.diagnose_sorting:
         st.set_config(dict(diagnose_sorting=True))
-    if args.config_from_json != '':
-        conf = json_to_dict(args.config_from_json)
-        st.set_config(conf)
+    if args.context_kwargs:
+        logging.info(f'set context kwargs {args.context_kwargs}')
+        st = getattr(module, args.context)(**args.context_kwargs)
```

Comment:
Are there use cases in which one has to explicitly overwrite certain context setting. Would not it be sufficient to use `st.new_context` here?

Reply:
No this is different as it also allows context (function) init kwargs like `use_rucio` which is not a context config